app.json

```diff
@@ -2,7 +2,7 @@
   "name": "Mozilla Foundation site",
   "description": "Mozilla Foundation site",
   "addons": [
-    "heroku-postgresql:mini"
+    "heroku-postgresql:basic --version=15"
```

Comment:
Bumping it to a basic plan will get rid of the pesky emails with `[warning] Database disruption imminent, row limit exceeded for mini database on Heroku app <app_name>`

Reply:
Worth a couple extra dollars, let's do it.